### ISSUE TITLE
Skittering Hollow and Voru'Kar

### ISSUE THREAD

**Issue by BurnedCrusader** [Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2#684]
**Describe your suggestion in full detail below:**
As it stands now there are no living Nerubians in Azeroth from 603 start onwards, only servants of Lich King, which wasn't fully true in lore. In Wow there can be found one Island named [Skittering Hollow](https://wowpedia.fandom.com/wiki/Skittering_Hollow), where kobolds, troggs, even trolls and nerubians live and try to control the island. These Nerubians are named [Voru'Kar](https://wowpedia.fandom.com/wiki/Voru%27kar) after their leader, [Vorus'Arak](https://wowpedia.fandom.com/wiki/Vorus%27arak). Vorus'Arak himself is the last spiderlord alive as it stands in lore now, so adding him as a challenging start would be nice. If not playing as him or anyone on the island Skittering Hollow may very well be used by murlocs or pirates. I suggest also adding two paths for him:
- After uniting the island under himself Vorus'Arak may start preparation for invasion back home on continent(ducal level cb on any title in Great Borean Tundra or Dragonblight). If while choosing this path player or the AI gets a hold of kingdom level title Vorus'Arak will get a prestige and piety boost, while getting kingdom invasion cb on any titles inside Northrend.
- After uniting the island Vorus'Arak may also choose to instead build up the island, giving some development boosts in it and allowing more building slots on the island capital county(Like vanila Isle of Man gets construction building slots if the pirate kingdom is formed). This path doesn't give special cb, but also may give piety and prestige when Vorus'Arak gets a kingdom level title

About where island itself should be placed I think of these 4 places, since they lack any islands and it will make sense to maybe add some here as break-aways from Northrend.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/59051051/133128070-b7cea858-fdca-4817-988a-ccd026bd204f.png)
I think the third and fourth one's are the most possible
</details>
Either island itself could be split into 3 different counties which make up a duchy.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/59051051/133129099-f8e42d2d-54b2-41e3-b86e-25359843ae51.png)
Red are Nerub controlled more mountainous part of the island, yellow are kobolds controlling two mountain lakes and probably some old Nerubian tunnels, and finally blue is either trolls or troggs hilly coast
</details>
Or we can add small islands doting sea around Skittering Hollow, since lots of island maps from WC3 may used

**Comment by ercarp** [comment]
This is a good suggestion, I would love to see Skittering Hollow added as a Nerubian-controlled island. It would be a good compromise for the lack of Azjol-Nerub.

**Comment by ercarp** [comment]
Visually, Skittering Hollow reminds me simultaneously of Sholazar Basin, Pandaria, and parts of Zandalar.

<details><summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/78575425/133182603-8b353b52-568d-4c61-9a16-8090a60f25b9.png)
![image](https://user-images.githubusercontent.com/78575425/133182619-f012c948-0a69-4136-b6ed-f47ab3bd266e.png)
![image](https://user-images.githubusercontent.com/78575425/133182627-28f6f0a6-f76a-407c-ab17-a0c2fd6b86d6.png)

</details>

I noticed the tall trees on Skittering Hollow were actually the same ones they used for Nazmir (these are called `monkeytrees` in WoW's game files), but I think this is more of a case of Blizzard wanting to reuse assets from the same expansion rather than an indication of where the island is located. I couldn't figure out where the other trees were from but I assume they were also used somewhere on Zandalar or Kul Tiras.

<details><summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/78575425/133182896-51b1948b-079d-476c-8a3f-ec2e867ba29e.png)

</details>

Overall, I'm fine with your location ideas and I think it should definitely be close to Northrend and not anywhere near Zandalar or Pandaria, but I think it could also make sense to place it west of Sholazar Basin due to the jungle-like terrain.

<details><summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/78575425/133184412-242ddb06-8cfc-4a49-afba-3a66615bef52.png)

</details>

**Comment by BurnedCrusader** [comment]
> but I think it could also make sense to place it west of Sholazar Basin due to the jungle-like terrain.

Yeah, that was a weird one, mostly cuz there were mogu too apparently, but I agree on the location. Still, the part I mentioned below lacks any islands and with terrain in that part of Northrend it seems strange imo. Maybe split this into issue about adding non-lore islands for geography's and gameplay sake? Some area's seem too...lacking
